examples/notebook/src/index.ts

```diff
@@ -16,7 +16,7 @@ import '@jupyterlab/theme-light-extension/style/theme.css';
 import '../index.css';
 
 import { ServiceManager } from '@jupyterlab/services';
-import { MathJaxTypesetter } from '@jupyterlab/mathjax2';
+import { MathJaxTypesetter } from '@jupyterlab/mathjax-extension';
```

Comment:
Looks like the new `MathJaxTypesetter` now takes a `JupyterFrontEnd` in its constructor, while the code below in this file still uses the previous initialization options.

Reply:
Sorry, but what exactly are these directories under `examples/`? I tried looking for references of this in the README.md and the online documentation, but can't find anything on this.

I need help retrieving a reference to a `JupyterFrontEnd` instance. I'm assuming this is a singleton, so I can't just instantiate a new one here. Usually I only work within the context of an extension, so that's just given to me as the first argument when `JupyterFrontEndPlugin.activate()` is called. It looks like in dev mode, `window.jupyterapp` is also available, but I'm not certain that these example directories are run in dev mode.

I'd be happy to send out a PR later adding a new `examples/README.md` file to help explain this to new developers like myself as well.

Reply:
Here is a simple solution:
1. Change `MathJaxTypesetter` constructor to take `commands: CommandRegistry` and update the initialization as needed
2. In example below pass `commands`

Reply:
Even better, remove the args in constructor and move them to plugin. I don't think the commands should be added in the typesetter itself.

Reply:
Fixed in latest revision.